core/templates/pages/topics-and-skills-dashboard-page/navbar/topics-and-skills-dashboard-navbar-breadcrumb.component.ts

```diff
@@ -13,21 +13,22 @@
 // limitations under the License.
 
 /**
- * @fileoverview Controller for the navbar breadcrumb of the collection editor.
+ * @fileoverview Component for the navbar breadcrumb of the collection editor.
  */
 
-require('domain/utilities/url-interpolation.service.ts');
+import { Component } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
 
-angular.module('oppia').directive('topicsAndSkillsDashboardNavbarBreadcrumb', [
-  'UrlInterpolationService', function(UrlInterpolationService) {
-    return {
-      restrict: 'E',
-      scope: {},
-      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-        '/pages/topics-and-skills-dashboard-page/navbar/' +
-        'topics-and-skills-dashboard-navbar-breadcrumb.directive.html'),
-      controller: [
-        function() {}
-      ]
-    };
-  }]);
+@Component({
```

Comment:
Follow the comments in the similar file above*

Reply:
Done